test/tracer_test.rb

```diff
@@ -170,7 +170,10 @@ def test_trace_all_args
     assert_equal('extra-resource', span.resource)
     assert_equal('my-type', span.span_type)
     assert_equal(yesterday, span.start_time)
-    assert_equal({ 'env' => 'test', 'temp' => 'cool', 'tag1' => 'value1', 'tag2' => 'value2' }, span.meta)
+    assert_equal('test', span.get_tag('env'))
+    assert_equal('cool', span.get_tag('temp'))
+    assert_equal('value1', span.get_tag('tag1'))
+    assert_equal('value2', span.get_tag('tag2'))
```

Comment:
mmm, here, to be truely conform to the previous test, we should also `assert_equal(4, span.meta.length)` to make sure no extra tag creep in. But this is a nitpick.